addons/mail/static/src/core/common/thread_service.js

```diff
@@ -300,15 +289,7 @@ export class ThreadService {
                 ...this.getFetchParams(thread),
                 around: messageId,
             });
-            thread.messages = messages.reverse().map((message) => {
-                if (message.parentMessage?.body) {
-                    message.parentMessage.body = markup(message.parentMessage.body);
-                }
-                return this.store.Message.insert({
-                    ...message,
-                    body: message.body ? markup(message.body) : message.body,
-                });
-            });
+            thread.messages = this.store.Message.insert(messages.reverse(), { html: true });
```

Comment:
Note: not from diff but it's really strange to receive something ordered by the server to immediately reverse it. Server should send it the correct way?